extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/config/KuduOptions.java

```diff
@@ -8,52 +8,28 @@
  */
 package org.locationtech.geowave.datastore.kudu.config;
 
+import com.beust.jcommander.Parameter;
 import org.locationtech.geowave.core.store.BaseDataStoreOptions;
-import org.locationtech.geowave.core.store.DataStoreOptions;
-import org.locationtech.geowave.core.store.StoreFactoryFamilySpi;
-import org.locationtech.geowave.core.store.StoreFactoryOptions;
-import org.locationtech.geowave.datastore.kudu.KuduStoreFactoryFamily;
 import org.locationtech.geowave.datastore.kudu.util.KuduUtils;
 
-public class KuduOptions extends StoreFactoryOptions {
-  protected BaseDataStoreOptions baseOptions = new BaseDataStoreOptions() {
-    @Override
-    public boolean isServerSideLibraryEnabled() {
-      return false;
-    }
-
-    @Override
-    protected int defaultMaxRangeDecomposition() {
-      return KuduUtils.KUDU_DEFAULT_MAX_RANGE_DECOMPOSITION;
-    }
-
-    @Override
-    protected int defaultAggregationMaxRangeDecomposition() {
-      return KuduUtils.KUDU_DEFAULT_AGGREGATION_MAX_RANGE_DECOMPOSITION;
-    }
-
-    @Override
-    protected boolean defaultEnableVisibility() {
-      return false;
-    }
-  };
-
-  public KuduOptions() {
-    super();
+public class KuduOptions extends BaseDataStoreOptions {
+  @Override
+  public boolean isServerSideLibraryEnabled() {
+    return false;
   }
 
-  public KuduOptions(final String geowaveNamespace) {
-    super(geowaveNamespace);
+  @Override
+  protected int defaultMaxRangeDecomposition() {
```

Comment:
Could we actually remove these overrides? These were copied over from another datastore, but there's no evidence that we need them for Kudu yet

Reply:
:+1: